src/languages/cpp.js

```diff
@@ -448,6 +521,7 @@ export default function(hljs) {
     // `return some()`, `else if()`, `(x*sum(1, 2))`
     variants: [
       {
+        returnBegin: true,
```

Comment:
We're trying to get away from these, please use a look-ahead instead to avoid capturing the content rather than `returnBegin`.

Reply:
I see.

Reply:
With v12 I'll add a note that this is deprecated in favor of literal lookahead or our `regex.lookahead` helper.